types.d.ts

```diff
@@ -44,6 +45,26 @@ interface HTMLBlock extends Node {
   };
 }
 
+interface Image extends Node {
```

Comment:
Since `Image` is already a type from `mdast`, do you think it'd be better to define this as `ReadmeImage` or some such?